pydantic/main.py

```diff
@@ -277,6 +292,7 @@ def is_untouched(v: Any) -> bool:
             ),
             '__schema_cache__': {},
             '__json_encoder__': staticmethod(json_encoder),
+            '__json_builtin_types_to_encode__': json_builtin_types,
```

Comment:
I'm not clear why this needs to be stored on the model. Can't `BUILTIN_TYPES` just be a constant that's used in `wrap_builtin_types`?

Reply:
It's more performance tweak. In case when there are no types that is subclass of a non-encodable then no wrapping will be performed.